src/Equinox.MessageDb/MessageDb.fs

```diff
@@ -292,12 +294,14 @@ type MessageDbContext(connection : MessageDbConnection, batchOptions : BatchOpti
         return Token.create version, Array.chooseV tryDecode events }
     member _.Reload(streamName, requireLeader, log, token, tryDecode)
         : Async<StreamToken * 'event array> = async {
-        let streamPosition = token.version + 1L
+        let tokenVersion = Token.version token
+        let streamPosition = tokenVersion + 1L
```

Comment:
maybe rename to `startPos` ? maybe add comment `// streamVersion (incusive) from which we want to load new events`